packages/web-react/src/components/Grid/useGridStyleProps.tsx

```diff
@@ -0,0 +1,54 @@
+import { ElementType, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import { SpiritGridProps } from '../../types';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+
+export interface GridStyles<T> {
+  /** className props */
+  classProps: string;
+  /** Props for the grid element. */
+  props: T;
+}
+
+const gridBreakpoints = ['mobile', 'tablet', 'desktop'];
```

Comment:
This is not just about the `Grid`, it's common breakpoints of our DS. Do you think we could store the constant somewhere higher in the source?

Reply:
Fixed in https://github.com/lmc-eu/spirit-design-system/pull/168